DTS-STN/Alpha-Site
# Fix minor content errors

# Description

[Content updates](https://trello.com/c/tzOFeWzX)

Fixed some errors on `/signup`, `/thankyou`, `/confirmation`, and `/unsubscribe`.

There is still a problem on `/error` with the ReportAProblem component that I'm still looking into.
![image](https://user-images.githubusercontent.com/34345435/129220065-db520cb7-63e7-43c2-8c48-3b887423ceb5.png)

Even though we pass the language in as a property, it doesn't display the correct language, and there is an error in the console:
![image](https://user-images.githubusercontent.com/34345435/129220248-e88010fb-fec3-4111-945e-041eaf97bd44.png)

I just wanted to get the other content updates in while I spend more time on this because I don't really know how long this bug will take to fix.

## Acceptance Criteria

The pages above match the content found in [Figma](https://www.figma.com/file/t5uDkPa86nsS3ssFESX2Em/BDM-DECD-Alpha-Site?node-id=242%3A3541)

## Test Instructions

1. Visit `/signup`, `/thankyou?ref=signup`,  `/thankyou?ref=unsubscribe`,  `/confirmation?ref=signup`,  `/confirmation?ref=unsubscribe`, and `/unsubscribe`
2. Verify that the text matches what is in the Figma designs

## Help Requested

If anyone knows anything about that bug, please let me know, I currently have no idea how to tackle it.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
